modules/mono/csharp_script.cpp

```diff
@@ -2108,6 +2100,8 @@ Error CSharpScript::reload(bool p_keep_state) {
 
 			script_class->fetch_methods_with_godot_api_checks(native);
 
+			load_script_signals(script_class);
+
```

Comment:
This needs to be added to `CSharpScript::create_for_managed_type(GDMonoClass *)` as well. Also I would prefer it before the previous statement or after the while loop, since those two are related.

Reply:
@neikeq alright I'm updating load_script_signals to accept another parameter to handle the `script->native`. Also moving load script below the while loop in both functions. compiling/testing now to make sure it works then ill push and rebase